HealthcareSystem/GraphicalEditor/Models/MapObjectRelated/MapObjectMetrics.cs

```diff
@@ -8,22 +8,18 @@ namespace GraphicalEditor.Models.MapObjectRelated
 {
     public class MapObjectMetrics
     {
-        private double _width;
-        private double _height;
-        private double _x;
-        private double _y;
+        public double WidthOfMapObject { get; set; }
+        public double HeightOfMapObject { get; set; }
+        public double XOfCanvas { get; set; }
+        public double YOfCanvas { get; set; }
 
-        public MapObjectMetrics(double x, double y, double width, double height)
+        public MapObjectMetrics(double xOfCanvas, double yOfCanvas, double widthOfMapObject, double heightOfMapObject)
         {
-            this.X = x;
-            this.Y = y;
-            this.Width = width;
-            this.Height = height;
+            XOfCanvas = xOfCanvas;
+            YOfCanvas = yOfCanvas;
+            WidthOfMapObject = widthOfMapObject;
+            HeightOfMapObject = heightOfMapObject;
         }
```

Comment:
These are much better names that completely conveys meanings.
I cannot overstate how easier it is to understand this code. 
Keep it up :)